### DISCUSSION TITLE
Does v1 support select model for subscribed users?

### DISCUSSION THREAD

**Discussion by lijiazheng99** [acheong08/ChatGPT#907] [General]
Does v1 have any parameter to select which model to use for subscribed users?

**Comment by acheong08** [comment]
set paid to true in config

**Reply by lijiazheng99** [reply]
Thanks a lot!! It worked.

**Reply by linhandev** [reply]
> set paid to true in config

Paid have two models available, model name in url: official explanation
- text-davinci-002-render-sha: Optimized for speed, currently available to Plus users. named default
- text-davinci-002-render-paid: The previous ChatGPT Plus model. named legacy

I didn't really notice substantial difference them
@acheong08

**Reply by acheong08** [reply]
The difference is that -sha is available to free users
